templates/cisco_ios_show_object-group.textfsm

```diff
@@ -0,0 +1,31 @@
+Value Required,Filldown OGR_TYPE (Service|Network)
+Value Required,Filldown OGR_NAME (\S+)
+Value DESCRIPTION (.+)
+Value NESTED_OGR (\S+)
+Value ANY (any)
+Value HOST (\d+\.\d+\.\d+\.\d+)
+Value HOST_RANGE_START (\d+\.\d+\.\d+\.\d+)
+Value HOST_RANGE_END (\d+\.\d+\.\d+\.\d+)
+Value NETWORK (\d+\.\d+\.\d+\.\d+)
+Value NETMASK (\d+\.\d+\.\d+\.\d+)
+Value PROTOCOL (\S+)
+Value PORT_MATCH (eq|neq|range|lt|gt)
+Value PORT ((?<!range\s)\S+)
+Value PORT_RANGE_START ((?<=range\s)\S+)
+Value PORT_RANGE_END (\S+)
+Value ICMP_TYPE (echo|echo-reply|administratively-prohibited|unreachable|port-unreachable|redirect|router-advertisement|router-solicitation|packet-too-big|time-exceeded|ttl-exceeded|parameter-problem)
+
+Start
+  ^(Service|Network) -> Continue.Clearall
+  ^${OGR_TYPE}\s+object\s+group\s+${OGR_NAME}\s*$$ -> Record
+  ^\s+Description\s+${DESCRIPTION}$$ -> Record
+  ^\s+group-object\s+${NESTED_OGR}\s*$$ -> Record
+  ^\s+(host\s+${HOST}|range\s+${HOST_RANGE_START}\s+${HOST_RANGE_END}|${ANY}|${NETWORK}\s+${NETMASK})\s*$$ -> Record
+  ^\s+icmp\s+${ICMP_TYPE}\s*$$ -> Record
+  ^\s+${PROTOCOL}\s+${PORT_MATCH}\s+(${PORT_RANGE_START}\s+${PORT_RANGE_END}|${PORT})\s*$$ -> Record
+  ^\s+${PROTOCOL}\s*$$ -> Record
+  ^\s*$$
+  ^.* -> Error "Could not parse line:"
```

Comment:
We typically just require ` ^. -> Error`

Reply:
```suggestion
  ^. -> Error
```
OK